commands/port.js

```diff
@@ -31,24 +36,33 @@ module.exports = function(topic, command) {
 
 function * run(context, heroku) {
   yield exec.initFeature(context, heroku, function *(configVars) {
-    let remotePort = context.args.port;
-    let localPort = context.flags.localPort || remotePort;
```

Comment:
@RasPhilCo is there a pattern for deprecating an arg? Like can we still accept it, but remove it from the auto-gen documentation?

Reply:
Make the deprecated arg hidden via `hidden: true`.